.travis.yml

```diff
@@ -22,7 +22,12 @@ jobs:
       os: linux  # No need to run this everywhere
       before_script:
         - travis_wait rustup install $(cat rust-toolchain)
-        - travis_wait rustup component add rustfmt-preview
```

Comment:
This is the documented way to install it on the rustfmt GitHub. Why are you changing it?

Reply:
I was confused why this setting doesn't complain when using different versions. I also still don't get which version is being used. Does it depend on the used toolchain ?

Reply:
It is my understanding that rustup handles keeping it in sync with the installed rust version.